docs/examples/getDataMoverResources.go

```diff
@@ -0,0 +1,217 @@
+package main
+
+import (
+	"fmt"
+	"os"
+	"os/exec"
+	"strings"
+
+	"github.com/spf13/cobra"
+)
+
+var (
+	backup  bool
+	restore bool
+	details bool
+)
+
+// TODO transform into flag?
+const namespace = "openshift-adp"
+
+// no --tty flag: Unable to use a TTY - input is not a terminal or the right kind of file
+const velero = "oc -n %v exec deployment/velero -c velero -i -- ./velero %v"
+
+var rootCmd = &cobra.Command{
+	Use: "getDataMoverResources",
+	Long: `Check the OADP DataMover CR's in real time.
+
+You need to log in to your cluster and deploy OADP prior to run this script`,
+	Args:    cobra.NoArgs,
+	Example: "go run docs/examples/getDataMoverResources.go -b",
+	Run:     rootRun,
+}
+
+func parseShellCommand(command string) *exec.Cmd {
+	commandAndArgs := strings.Split(command, " ")
+	return exec.Command(commandAndArgs[0], commandAndArgs[1:]...)
+}
+
+func runShellCommand(command string) error {
+	cmd := parseShellCommand(command)
+	return cmd.Run()
+}
+
+func captureShellOutput(command string) string {
+	cmd := parseShellCommand(command)
+	stdout, err := cmd.CombinedOutput()
+
+	// TODO should we ignore errors?
+	if err != nil {
+		fmt.Print(string(stdout))
+		// TODO can't we get the return code from the command run?
+		os.Exit(1)
+	}
+
+	return string(stdout)
+}
+func captureShellOutputA(command []string) string {
+	cmd := exec.Command(command[0], command[1:]...)
+	stdout, err := cmd.CombinedOutput()
+
+	// TODO should we ignore errors?
+	if err != nil {
+		fmt.Print(string(stdout))
+		// TODO can't we get the return code from the command run?
+		os.Exit(1)
+	}
+
+	return string(stdout)
+}
+
+func getNumberOfMatches(lines string, match string) int {
+	// | grep -c "match"
+	return strings.Count(lines, match)
+}
+
+func getNumberOfResources(lines string) int {
+	// | sed 1d | wc -l
+	numberOfMatches := getNumberOfMatches(lines, "\n") - 1
+	if numberOfMatches > 0 {
+		return numberOfMatches
+	}
+	return 0
+}
+
+func getLinesOfMatches(lines string, match string) string {
+	array := strings.Split(lines, "\n")
+	matches := ""
+	for _, line := range array {
+		if strings.Contains(line, match) {
+			matches = matches + line
+		}
+	}
+	return matches
+}
+
+func rootRun(cmd *cobra.Command, args []string) {
+	// this checks slow the script
```

Comment:
I wanted to add validations, but the way the script is intended to run I think it is not good. With `watch`, it would run this every iteration. Better to remove, right?